fv3core/stencils/dyn_core.py

```diff
@@ -132,29 +136,36 @@ def p_grad_c_stencil(
             )
 
 
-def get_n_con():
-    if spec.namelist.convert_ke or spec.namelist.vtdm4 > 1.0e-4:
-        n_con = spec.grid.npz
+def get_nk_heat_dissipation(namelist, grid):
```

Comment:
A comment for devs would be useful here based on what you told me earlier.

```suggestion
def get_nk_heat_dissipation(namelist, grid):
    # determines whether to convert dissipated kinetic energy into heat in the full
    # column, not at all, or in 1 or 2 of the top of atmosphere sponge layers
```